### PR TITLE
TorrentLeech - Updated Categories from website

### PR DESCRIPTION
**Pull request by blastcodem**
Added new naming for new categories
Added to option to new categories name (Kept old one for backward compatiblity)

I can update wiki when pulled

### CODE REVIEW

**Comment by cvium** [comment]
> Added to option to new categories name (Kept old one for backward compatiblity)

Are they still using the old categories?

File: flexget/plugins/sites/torrentleech.py [hunk 1]
```diff
@@ -23,12 +23,23 @@
     # Movies
     'Cam': 8,
     'TS': 9,
+    'TS/TC': 9,
+#    R5 is deprecated
     'R5': 10,
     'DVDRip': 11,
+	'DVDRip/DVDScreener': 11,
```
Comment (cvium) [inline flexget/plugins/sites/torrentleech.py]: What's with all the random indentations?

Comment (blastcodem) [reply flexget/plugins/sites/torrentleech.py]: Copy paste, Error, will fix

File: flexget/plugins/sites/torrentleech.py [hunk 2]
```diff
@@ -48,9 +59,10 @@ class UrlRewriteTorrentleech(object):
           password: xxxxxxxx  (required)
           category: HD
 
-          Category is any combination of: all, Cam, TS, R5,
-          DVDRip, DVDR, HD, BDRip, Movie Boxsets, Documentaries,
-          Episodes, TV BoxSets, Episodes HD
+          Category is any combination of: all, Cam, TS, TS/TC, DVDRip,
+				DVDRip/DVDScreener, WEBRip': 37, HDRip': 43, BDRip, DVDR,
```
Comment (cvium) [inline flexget/plugins/sites/torrentleech.py]: This doesn't look right either

**Comment by cvium** [comment]
I see no reason to keep any categories they no longer support

**Pull request merged**